cloud-controller-manager/do/certificates.go

```diff
@@ -0,0 +1,7 @@
+package do
```

Comment:
You need to add the licence header to this file as well.

Reply:
doh, thanks for the reminder